Externals/FreeSurround/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+set(SRCS
+	ChannelMaps.cpp
+	KissFFT.cpp
+	KissFFTR.cpp
+	_KissFFTGuts.h
+	FreeSurroundDecoder.cpp
+	)
+
+add_library(FreeSurround STATIC ${SRCS})
```

Comment:
Put: target_include_directories(FreeSurround PUBLIC .) here to do the same as before, but better.

BUT! Using just "Externals" means you could include any other headers from that folder by linking this target and propagating the include directories. This breaks isolation.

It also means that all the other includes should be changed from "FreeSurround/foo.h" to just "foo.h". Alternatively, you could put the public API headers in a folder "Externals/FreeSurround/include/FreeSurround" and just add the folder "include" and keep the same include directives you currently have.